mathesar_ui/src/pages/schema/SchemaPage.svelte

```diff
@@ -59,7 +59,9 @@
   $: tablesMap = canExecuteDDL ? $tablesStore.data : $importVerifiedTablesStore;
   $: explorationsMap = $queries.data;
   $: isTablesLoading = $tablesStore.state === States.Loading;
+  $: isTablesUnfetchable = $tablesStore.state === States.Error;
   $: isExplorationsLoading = $queries.requestStatus.state === 'processing';
+  $: isExplorationsUnfetchable = $queries.requestStatus.state === 'failure';
```

Comment:
instead of passing separate variables for loading & error, please use `RequestStatus` (refer: mathesar_ui/src/api/utils/requestUtils.ts). This will let you pass error message too to the child component. 
You can then use the `error` string to show on the UI instead a generic message("Error fetching tables") from Figma. This give more information to the user and help him/her to report better errors .